lib/active_record_shards/tasks.rb

```diff
@@ -77,3 +82,15 @@
     end
   end
 end
+
+module ActiveRecordShards
+  module Tasks
+    def self.env_name
+      ActiveRecordShards.rails_env || 'development'
```

Comment:
maybe `ActiveRecordShards.rails_env` could already include the `|| 'development'`